giskard/commands/cli_worker.py

```diff
@@ -104,8 +118,11 @@ def start_command(url: AnyHttpUrl, is_server, api_key, is_daemon, hf_token, nb_w
     """
     analytics.track(
         "giskard-worker:start",
-        {"is_server": is_server, "url": anonymize(url), "is_daemon": is_daemon},
+        {"is_server": is_server, "url": anonymize(url), "is_daemon": is_daemon, "log_level": log_level},
     )
+
+    logging.root.setLevel(logging.getLevelName(log_level))
```

Comment:
Setting log level does not work for me : if I set it to error, I still have logs from giskard weirdly. Not sure why

Reply:
Normally setting Error level to root level should suppress every log even from giskard, in my case worker is still starting with logs.

Reply:
Nvm, it was only at start, seems ok after